app/src/lib/git/core.ts

```diff
@@ -347,3 +347,18 @@ export async function gitNetworkArguments(
 export function parseCommitSHA(result: IGitResult): string {
   return result.stdout.split(']')[0].split(' ')[1]
 }
+
+/**
+ * Run a Git command and return whether the exit code indicated success
+ *
+ * This defers to the default error handling infrastructure inside if an error
+ * is encountered.
```

Comment:
> default error handling infrastructure inside

I wasn't sure what this meant. Does that mean inside this method, or inside something else?